app/views/announcements/_announcement.html.erb

```diff
@@ -0,0 +1,8 @@
+  <% if announcement_exists?(current_announcement) %>
+    <% if current_announcement.start_time <= Date.current && current_announcement.end_time >= Date.current %>
+      <div class=" alert alert-success  container text-center">
+        <p><%== current_announcement.body %></p>
```

Comment:
Also add a optional field for clickable links

Reply:
I didn't got it what exactly is to be done ....

Reply:
didn't got it what exactly has to be done...

Reply:
If an announcement has a link(CTA), then add it here, also add in the db

Reply:
optional field if annnouncement posses any relevant link
![2020-10-15_19-33](https://user-images.githubusercontent.com/55848322/96141251-71f13200-0f1e-11eb-8876-0511463c0edb.png)

displaying only if it posses link other wise only announcement
![2020-10-15_19-44](https://user-images.githubusercontent.com/55848322/96141746-f0e66a80-0f1e-11eb-8f62-cca9fb41a4fc.png)

![2020-10-15_19-35](https://user-images.githubusercontent.com/55848322/96141227-6e5dab00-0f1e-11eb-9339-f93b77f980ef.png)

Reply:
yup not worked on ui yet wanted to ensure correct backend functionality first will  some good designs soon for the review

Reply:
Tried to use high contrast  compliment colors to grab attentions easily...

![v11](https://user-images.githubusercontent.com/55848322/96240193-3ad25d80-0fbe-11eb-8a28-b625301e4b78.png)
[v2](https://user-images.githubusercontent.com/55848322/96239825-cd263180-0fbd-11eb-9c96-b34a5043371f.png)
![2020-10-16_14-40](https://user-images.githubusercontent.com/55848322/96239839-d0212200-0fbd-11eb-8e96-e508df35a1a0.png)
![v4](https://user-images.githubusercontent.com/55848322/96240208-402fa800-0fbe-11eb-93d9-3bee4d13b2ad.png)

Reply:
also title of clickable links must be fixed ? (like used here ```read more ->```)